cmd/client/BUILD

```diff
@@ -17,6 +12,6 @@ go_library(
 )
 
 go_binary(
-    name = "client",
```

Comment:
This isn't working well with gazelle, that's why @kyessenov made this.

Please mark `# gazelle:ignore` at the top of this file (and other BUILD files in `cmd`).